src/main/java/net/laboulangerie/laboulangeriecore/misc/KingCondition.java

```diff
@@ -0,0 +1,22 @@
+package net.laboulangerie.laboulangeriecore.misc;
+
+import java.util.UUID;
+
+import com.palmergames.bukkit.towny.TownyUniverse;
+import com.palmergames.bukkit.towny.object.Resident;
+
+import pl.betoncraft.betonquest.Instruction;
+import pl.betoncraft.betonquest.api.Condition;
+import pl.betoncraft.betonquest.exceptions.QuestRuntimeException;
+
+public class KingCondition extends Condition {
+    @SuppressWarnings("deprecation")
+    public KingCondition(Instruction instruction) {
+        super(instruction);
+    }
+    @Override
+    protected Boolean execute(String playerID) throws QuestRuntimeException {
+        Resident resident = TownyUniverse.getInstance().getResident(UUID.fromString(playerID));
```

Comment:
Useless temp variable you could just `return TownyUniverse.getInstance().getResident(UUID.fromString(playerID)).isKing();`

Reply:
In fact I should check if resident doesn't equal null